extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1202,4 +1203,26 @@ describes.realWin('amp-story', {
           });
     });
   });
+  describe('amp-story branching', () => {
+    it('should navigate to the target page when a goToPage action is executed',
+        () => {
+          createPages(story.element, 4,
+              ['cover', 'page-1', 'page-2', 'page-3']);
+          toggleExperiment(win, 'amp-story-branching', true);
+          story.buildCallback();
+          return story.layoutCallback()
+              .then(() => {
+                story.element.setAttribute('id', 'story');
+                const actionButton = createElementWithAttributes(
+                    win.document,
+                    'button',
+                    {'id': 'actionButton',
+                      'on': 'tap:story.goToPage(id=page-2)'});
+                element.querySelector('#cover').appendChild(actionButton);
+                // Click on the actionButton to trigger the goToPage action.
+                actionButton.click();
```

Comment:
can you verify that the story is now on `page-2`?

`expect(story.activePage_.element.id).to.equal('page-2');`

Reply:
Thanks! Great work :)